cpp/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2017 Snowflake Computing, Inc. All rights reserved.
+#
+# CMakeList for Snowflake Client++
+#
+
+set(SOURCE_FILES_CXX
+        include/snowflake++/SnowflakeInclude.hpp
+        include/snowflake++/SnowflakeConnection.hpp
+        include/snowflake++/SnowflakeStatement.hpp
```

Comment:
Also the header file name should be compact, e.g., Util.hpp, Connection.hpp and Statement.hpp as the directory name already have `snowflake` name space.